maintainer/channels.py

```diff
@@ -114,13 +105,11 @@ def delete_children(self):
                             cid]["channel_topic"] == 'quarantine':
                     requests.delete('{}/channels/{}'.format(
                         self.connection, cid), auth=(self.username, self.password))
-                    return "deleted children channel {}".format(cid)
+                    return logging.log("deleted children channel {}".format(cid))
```

Comment:
What does log method return? I think this will be None?

Reply:
fixed logging